src/core/core.tooltip.js

```diff
@@ -484,6 +484,22 @@ var exports = module.exports = Element.extend({
 		this._lastActive = [];
 	},
 
+	// Dispatch mouse event on current mouse position
+	// Called in Chart.update()
+	dispatchMouseEvent: function() {
```

Comment:
Instead of dispatching a new event and thus go through the event loop (again?) and potentially trigger an additional `render()`, what about caching the whole event and call handleEvent() again in this method (that I would rename to `_refresh()` (private)):

```javascript
_refresh: function() {
    if (this._lastEvent) {
        this.handleEvent(this._lastEvent);
    }
}
```

We should also avoid dispatching platform specific events since this should be abstracted by each platform implementations.

Reply:
I will give this a go!